### PR TITLE
Chiorean Rebeca project documentation

### PR DESCRIPTION
**Pull request by ChioreanRebeca**
### Pull Request Overview

This pull request adds the documentation for the Pico Wand project.

### TODO or Help Wanted

Please let me know if there is anything to change.

### Author

Signed-off-by: Chiorean Rebeca <chioreanrebeca01@gmail.com>

### CODE REVIEW

**Review by alexandruradovici — APPROVED** [review]

**Comment by ChioreanRebeca** [comment]
i hope this solved the issue :)

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png
Comment (alexandruradovici) [inline docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: Rename the folder using your username.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png
Comment (ChioreanRebeca) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: I used my curs.upb username as mentioned on Teams " your_curs.upb.ro_username ". Should I change the folder name to my GitHub username?

Comment (alexandruradovici) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: Is your username *Rebecca_CHIOREAN*, spelled like this?

Comment (ChioreanRebeca) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: No, my username is Rebeca CHIOREAN (with just one "c").

Comment (alexandruradovici) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: Rhe username cannot be with spaces, it is the part before the @ of your email.

Comment (ChioreanRebeca) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: Ok I undestarnd now. Username is rebeca.chiorean with small letters.

Comment (ChioreanRebeca) [reply docs/project/2024/Rebeca_CHIOREAN/architectureChioreanLita.png:1]: I hope it is okay now.

File: docs/project/2024/rebeca.chiorean/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Pico Wand
+:::info
```
Comment (alexandruradovici) [inline docs/project/2024/rebeca.chiorean/index.md:2]: One more thing, please add a one line description koi the project here.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**